<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatLibForm</class>
 <widget class="QWidget" name="MatLibForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/images/images/db.png</normaloff>:/images/images/db.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Material name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mat_name">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Metal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Composite</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Material</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Properties</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>Permalloy 81H</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Metal</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>εr=1 μr=10000, σr=12</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Steel</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>Metal</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>εr=1 μr=180, σr=10</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Magnesium MA2-1</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>Metal</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>εr=1 μr=1, σr=8.3</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>Aluminum</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>Metal</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>εr=1 μr=1, σr=37</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Copper</string>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>Metal</string>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>εr=1 μr=1, σr=57</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>Z-type Hexaferrite</string>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string>Composite</string>
        </property>
       </item>
       <item row="5" column="2">
        <property name="text">
         <string>εr=9 μr=2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QTextBrowser" name="descript_browser">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;εr=1 μr=10000, σr=12&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="load_but">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Properties</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="new_but">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
